### PR TITLE
add openSync function definition to ibm_db package

### PR DESCRIPTION
**Pull request by bensonma**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ibmdb/node-ibm_db/blob/master/APIDocumentation.md#openSyncApi
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ibm_db ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ibm_db@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'ibm_db'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2b3d1b8aa07d12410426be34e8dc28407862bf32
</p>

File: types/ibm_db/index.d.ts [hunk 1]
```diff
@@ -222,6 +222,7 @@ export class Pool implements PoolOptions {
   usedPool: object;
   poolsize: number;
   odbc: ODBC;
+  openSync(connStr: string): Database;
```
Comment (amcasey) [inline types/ibm_db/index.d.ts:225]: Should it have an optional `options` parameter?

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/ibm_db/index.d.ts (hunk 1 above)
Comment (bensonma) [reply types/ibm_db/index.d.ts:225]: No. The Pool.prototype.openSync has only the one connStr param.

See: 
https://github.com/ibmdb/node-ibm_db/blob/ad38da18e56deee23288ba42a3092134c32b0456/lib/odbc.js#L2113

Comment (amcasey) [reply types/ibm_db/index.d.ts:225]: Weird - unless I'm misreading, the doc and sample show options.

**Review by amcasey — APPROVED** [review]

File: types/ibm_db/index.d.ts (hunk 1 above)
Comment (amcasey) [reply types/ibm_db/index.d.ts:225]: Oh, the docs are for https://github.com/ibmdb/node-ibm_db/blob/ad38da18e56deee23288ba42a3092134c32b0456/lib/odbc.js#L146.

**Comment by bensonma** [comment]
Ready to merge

**Pull request merged**